import-scripts/update-az-mskimpact.sh

```diff
@@ -0,0 +1,141 @@
+#!/bin/bash
+
+export AZ_DATA_HOME=$PORTAL_DATA_HOME/az-msk-impact-2022
```

Comment:
We need to know whether we are keeping "running repo" which will get old history purged ... or whether we are "rolling over" the repo each year / time period into a new/empty/fresh repo created for the new time period.

Reply:
let's use a symlink .. just like dmp->dmp-2022